core/src/main/scala/filodb.core/memstore/PartKeyLuceneIndex.scala

```diff
@@ -646,7 +648,7 @@ class PartKeyLuceneIndex(ref: DatasetRef,
       s"with startTime=$startTime endTime=$endTime into dataset=$ref shard=$shardNum")
     val doc = luceneDocument.get()
     val docToAdd = doc.facetsConfig.build(doc.document)
-    val term = new Term(PART_ID, documentId)
+    val term = new Term(PART_ID_STRING, documentId)
```

Comment:
Unrelated to your PR, but can you please improve by adding docs on why documentId is assigned to partId ? @amolnayak311  can add more here, but this should be called only from DownsampleTimeSeriesShard where there is no concept of partId. Hence documentId is used.

Reply:
> but this should be called only from DownsampleTimeSeriesShard where there is no concept of partId.

Hi @vishramachandran regarding above, the upsert is called from any ``*TimeSeriesShard``, however the default parameter ``documentId`` is only overridden from ``DownSampleTimeSeriesShard`` as you mentioned. Initially it was assumed to be the string representation of partId to uniquely identify the document to upsert, however, in downsample index, we still need a unique identifier that wasn't a numeric value (which was ephemeral) hence the method was amended to optionally accept that unique identifier.

Reply:
I like the `PART_ID` prefix because both are really just partition identifiers by different names. Either way works for me, though -- I will change it if there's a strong preference for `DOCUMENT_ID`.